examples/composite_template/ex_application_window/imp.rs

```diff
@@ -0,0 +1,51 @@
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use gtk::{glib, CompositeTemplate};
+
+/// The private struct, which can hold widgets and other data.
+#[derive(Debug, Default, CompositeTemplate)]
+#[template(file = "ex_application_window.ui")]
+pub struct ExApplicationWindow {
+    // The #[template_child] attribute tells the CompositeTemplate macro
+    // that a field is meant to be a child within the template.
+    #[template_child]
+    pub headerbar: TemplateChild<gtk::HeaderBar>,
+    #[template_child]
+    pub label: TemplateChild<gtk::Label>,
+    // You can specify the optional `id` argument if the id is not the same
+    // as the identifier
+    #[template_child(id = "subtitle_label")]
+    pub subtitle: TemplateChild<gtk::Label>,
+    #[template_child]
+    pub menubutton: TemplateChild<crate::ex_menu_button::ExMenuButton>,
```

Comment:
```suggestion
    pub menubutton: TemplateChild<ExMenuButton>,
```
& use `use crate::ex_menu_button::ExMenuButton`; maybe?

Reply:
Fixed